docs/tutorials/deployments.md

```diff
@@ -191,18 +186,30 @@ You can list the contents of the folder to see what we have at this step in the
 <div class="terminal">
 ```bash
 ~/prefect-tutorial $ ls -a
-./                        .prefectignore            log_flow-deployment.yaml
-../                       __pycache__/              log_flow.py
+.				.prefectignore		config.json			        log_flow.py
+..				__pycache__			log_flow-deployment.yaml	utilities.py
 ```
 </div>
 
-## Configure the deployment
+???+ tip "Ignoring files with `.prefectignore`"
```

Comment:
The `???+` makes it a collapsible block that is by default fully expanded until the user clicks to collapse it.  Since it was such a big block I wanted to test out that functionality (documented [here](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#collapsible-blocks)).

Let me know what you think, I can change it back

Reply:
Oh that's cool! I knew this was possible, but had not tried it yet. Very good use of this feature. I's okay as is then